packages/cli-kit/src/public/node/system.ts

```diff
@@ -133,9 +134,13 @@ export async function sleep(seconds: number): Promise<void> {
  * will be used.
  *
  * @param stdin - The standard input stream to check.
+ * @param env - Environmemnt variables.
  * @returns True in the selected input stream support raw mode.
  */
-export function terminalSupportsRawMode(stdin?: ReadStream): boolean {
+export function terminalSupportsRawMode(stdin?: ReadStream, env = process.env): boolean {
+  if (isTruthy(env.CI)) {
```

Comment:
Is this `CI` env-var set for all CI platforms?

Reply:
Yes, its as close to a standard as something like this gets